torchmultimodal/modules/layers/attention.py

```diff
@@ -85,7 +85,12 @@ def _combine_multihead(self, x: Tensor) -> Tensor:
         return shift_dim(x, 1, -2).flatten(start_dim=-2)
 
     def forward(
-        self, q: Tensor, k: Tensor, v: Tensor, decode_step=None, decode_idx=None
+        self,
+        q: Tensor,
+        k: Tensor,
+        v: Tensor,
+        decode_step: int = None,
+        decode_idx: Iterable[int] = None,
```

Comment:
I would do this: 
```
decode_step: Optional[int] = None,
decode_idx: Optional[List[int]] = None,
```
this is actually a change I've made in a separate PR: https://github.com/facebookresearch/multimodal/pull/101/files